packages/dropdown/src/presenters/OptionPresenter.js

```diff
@@ -84,14 +84,24 @@ export default class OptionPresenter extends Component {
     const { children, selected, ...otherProps } = this.props;
 
     return (
-      <OptionWrapper selected={selected} {...otherProps}>
-        <span>{children}</span>
-        {selected && (
-          <div>
-            <CheckmarkBlueDark24 color="#087AAD" />
-          </div>
-        )}
-      </OptionWrapper>
+      <ThemeContext.Consumer>
+        {({ metadata }) => {
+          const Icon =
+            metadata.densityId === "medium-density"
+              ? CheckmarkSUI
+              : CheckmarkXsUI;
+          return (
+            <OptionWrapper selected={selected} {...otherProps}>
+              <span>{children}</span>
+              {selected && (
+                <div>
+                  <Icon color="#087AAD" />
```

Comment:
- As icon `CheckmarkBlueDark24` has been removed in Icons 3.0.0, I am using `CheckmarkSUI` / `CheckmarkXsUI` for medium / high density for the selected option, I don't find Checkmark**M**UI Is this ok? or it's designed to use Checkmark24? 

- In the old code, the option selected icon was colored by hard code as `#087AAD`, i don't find this color in HIG colors, I am thinking if we should update this to use themeData  color like `colorScheme.accentColor`? (`colorScheme.accentColor` is also used as dropdown caret icon's cover color )

Reply:
I think CheckmarkSUI is perfect for this. I think the hard coding of `#087AAD` is fine for now as well as we are going to be creating a `menu` component soon. From that we will be replacing this with that component which should be using theme-data.